### PR TITLE
Explicitly reference content-type and content-encoding in the proposal introduction.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Explicitly reference content-type and content-encoding in the proposal introduction.

@LPardue does it sound better? Actuallly the method's too impacts on
resource representation (eg. HEAD vs GET). Should we state it here too?

### CODE REVIEW

**Comment by LPardue** [comment]
how about this wording?

```
The value of `Digest` is calculated on selected representation, which is tied to the value contained in any `Content-encoding` or `Content-type` header fields. Therefore, a given resource may have multiple different digest values.
```

**Comment by ioggstream** [comment]
@LPardue Question:

1- does `selected representation` identify only responses values tied to the `Accept-*` headers and content-negotiation?
2- does `selected representation` identify requests too (eg. where the selection is not actually negotiated)?

In the second case, your wordings is fine.
